Errors while building wheel on Windows

```
module.c(508): error C2440: 'function': cannot convert from 'double' to 'const char *'
module.c(508): warning C4024: 'PyModule_AddStringConstant': different types for formal and actual parameter 3
module.c(508): error C2143: syntax error: missing ')' before 'constant'
module.c(508): error C2059: syntax error: ')'
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Professional\\VC\\Tools\\MSVC\\14.15.26726\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
Failed building wheel for ciso8601
```

Python 3.7.0 x64
VS Professional 2017 (15.8.5)
Windows 7 SP1 x64

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,7 +2,7 @@
 <!-- TOC anchorMode:github.com -->
 
 - [2.x.x](#2xx)
-    - [Version 2.1.1 (Unreleased)](#version-211-unreleased)
+    - [Version 2.1.1](#version-211)
     - [Version 2.1.0](#version-210)
     - [Version 2.0.1](#version-201)
     - [Version 2.0.0](#version-200)
@@ -17,9 +17,9 @@
 
 # 2.x.x
 
-## Version 2.1.1 (Unreleased)
+## Version 2.1.1
 
-* Fixed a problem where builds on Windows were not working (#76).
+* Fixed a problem where builds on Windows were not working (#76). Thanks @alexandrul and @gillesdouaire, and sorry.
 
 ## Version 2.1.0
 

--- a/setup.py
+++ b/setup.py
@@ -30,7 +30,7 @@ if os.environ.get("STRICT_WARNINGS", '0') == '1':
         os.environ['_CL_'] = ""
     os.environ['_CL_'] += " /WX"
 
-VERSION = "2.1.0"
+VERSION = "2.1.1"
 
 setup(
     name="ciso8601",

